lua/xcodebuild/helpers.lua

```diff
@@ -72,8 +72,15 @@ end
 ---@return table<string, string[]>
 function M.find_all_swift_files()
   local util = require("xcodebuild.util")
-  local allFiles =
-    util.shell("find '" .. vim.fn.getcwd() .. "' -type f -iname '*.swift' -not -path '*/.*' 2>/dev/null")
+
+  local allFiles
+  if util.is_fd_installed() then
+    allFiles = util.shell("fd -I '.*\\.swift' " .. vim.fn.getcwd() .. " --type f 2> /dev/null")
```

Comment:
When using `fd` you must add `$` at the end of each pattern, otherwise it also matches other files like `swiftinterface`